src/utils/node.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Fluidware srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function ensureError(value: unknown): Error {
+  if (value instanceof Error) return value;
+
+  let stringified = '[Unable to stringify the thrown value]';
+  try {
+    stringified = JSON.stringify(value);
+  } catch {
+    // doesn't matter
+  }
+
+  return new Error(`This value was thrown as is, not through an Error: ${stringified}`);
+}
+
+export function getEnvironment() {
+  return process.env;
+}
+
+/* eslint-disable no-process-env */
```

Comment:
maybe we want this plugin enabled within this repo?

Reply:
indeed